client/src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, Layout } from 'antd';
+import React, { Fragment } from 'react';
+
+import styles from './Sidebar.module.css';
+import SidebarItem from './SidebarItem';
+
+const { Header, Footer } = Layout;
+
+const bookmarks = [
+  { name: 'Strona Główna', url: '#home' },
+  { name: 'Przeliczone puszki', url: '#count' },
+  { name: 'Administracja', url: '#admin' },
+  { name: 'Puszki', url: '#box' },
+];
+
+const Sidebar: React.FC = (props) => {
+  const userName = 'superadmin';
+  const isLogin = true;
+
+  return (
+    <Fragment>
```

Comment:
Would be good idea to reformat this code into two components, due to low readibility. E.g. <ProtectedContent /> and <UnprotectedContent />

Reply:
Github has eaten part of my comment. Here's how it should have been: E.g. `<ProtectedContent />` and `<UnprotectedContent />`.